### ISSUE TITLE
Error: Cannot find module './arc', version 3.13.1

### ISSUE THREAD

**Issue by aripekkako** [Turfjs/turf#615]
Hi, when trying to require the "@turf/turf" module in Node.js I get the following error:

```
module.js:327
    throw err;
    ^

Error: Cannot find module './arc'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (****/node_modules/@turf/great-circle/index.js:1:73)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)

```

To reproduce simply put the following code into a file:

```
const turf = require('@turf/turf');
```

and then run the execute the file after npm installing with the following package.json dependecies:

```
  "dependencies": {
    "@turf/turf": "^3.13.1"
  }
```

Tested with Node.js 4.8.0, 6.2.2 and 6.9.1.

**Comment by DenisCarriere** [comment]
👍 Thanks for reporting this, should be fixed now with `3.13.2`.

@aripekkako

**Comment by DenisCarriere** [comment]
Should be fixed by now.
